src/e2e-test/features/salesforcebatchsource/RunTime.feature

```diff
@@ -126,3 +126,42 @@ Feature: Salesforce Batch Source - Run time Scenarios
     And Open and capture logs
     And Verify the pipeline status is "Succeeded"
     Then Close the pipeline logs
+
+  @BATCH-TS-SF-RNTM-04 @BQ_SINK_TEST @BQ_TEMP_CLEANUP @CREATE_CUSTOM @DELETE_OBJECT
```

Comment:
There are test cases above which are already running pipeline for salesforce to BQ no need to write a new tests. Add the new hooks and data validation to the above tests.

Reply:
done